File: app/src/main/java/tv/superawesome/demoapp/MyApplication.java
package tv.superawesome.demoapp;

import androidx.multidex.MultiDexApplication;

import tv.superawesome.sdk.publisher.AwesomeAds;

/**
 * Created by gabriel.coman on 30/04/2018.
 */

public class MyApplication extends MultiDexApplication {

    @Override
    public void onCreate() {
        super.onCreate();
        AwesomeAds.init(this, true);
    }
}

File: app/src/main/java/tv/superawesome/demoapp/MyApplication.kt
package tv.superawesome.demoapp

import androidx.multidex.MultiDexApplication
import tv.superawesome.lib.sabumperpage.SABumperPage
import tv.superawesome.sdk.publisher.common.models.Configuration
import tv.superawesome.sdk.publisher.common.network.Environment
import tv.superawesome.sdk.publisher.common.sdk.AwesomeAds
import tv.superawesome.sdk.publisher.common.ui.common.BumperPage

class MyApplication : MultiDexApplication() {
    var environment: Environment = Environment.Production
        private set

    override fun onCreate() {
        super.onCreate()

        BumperPage.overrideName("AwesomeAds Demo")
        SABumperPage.overrideName("AwesomeAds Demo")

        AwesomeAds.init(
            this,
            Configuration(logging = true, environment = environment)
        )
        tv.superawesome.sdk.publisher.AwesomeAds.init(this, true)
    }

    fun setupUITest() {
        environment = Environment.UITesting
    }
}
